### ISSUE TITLE
Documentation links return a 404 error

### ISSUE THREAD

**Issue by gelliott181** [forcedotcom/SalesforceMobileSDK-iOS#2631]
Please fill out the following details:

1. Version of Mobile SDK Used: 6.2.0
2. Issue found in Native App or Hybrid App: Some documentation links in the README lead to non-existent pages
3. OS Version: GitHub
4. Device: GitHub
5. Steps to reproduce: Attempt to access the SalesforceSwiftSDK Documentation from the README in the repo
6. Actual behavior: 404 Not Found
7. Expected Behavior: Documentation loads with a success response
8. Error Log: 
```404
File not found

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL.
For root URLs (like http://example.com/) you must provide an index.html file.

Read the full documentation for more information about using GitHub Pages.
```

**Comment by gelliott181** [comment]
I believe the readme needs an audit. The build documentation (Current link: https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/master/build.md) is not at the linked location of the repo either.

**Comment by wmathurin** [comment]
We will fix that in the next release.
Thanks for bringing it to our attention.

**Comment by wmathurin** [comment]
@rwhitley

**Comment by wmathurin** [comment]
Fixed in dev (will be in Mobile SDK 7.0).
